client/src/Movies/Movie.js

```diff
@@ -11,7 +13,7 @@ export default class Movie extends Component {
 
   componentDidMount() {
     // change this line to grab the id passed on the URL
-    const id = 1;
+    const id = this.props.match.params.id;
```

Comment:
You can use destructuring here to cut down on the redundancy of 'id' like so: `const { id } = this.props.match.params;`